mockery/generator.go

```diff
@@ -631,6 +631,8 @@ func (g *Generator) Write(w io.Writer) error {
 
 	res, err := imports.Process("mock.go", theBytes, opt)
 	if err != nil {
+		line := "--------------------------------------------------------------------------------------------"
+		fmt.Fprintf(os.Stderr, "Between the lines is the file (mock.go) mockery generated in-memory but detected as invalid:\n%s\n%s\n%s\n", line, g.buf.String(), line)
```

Comment:
I didn't know what `mock.go` was until looking in the source. The errors that I've encountered at that site have all mentioned `mock.go`. I couldn't see what's inside `mock.go` to associate line numbers with the source that caused the error.

The whole motivation for this was ambiguity/opaqueness. I'm comfortable if this change swings to the other end and appears to over-communicate.